axelrod/tests/unit/test_resultset.py

```diff
@@ -1,18 +1,23 @@
-import csv
 import unittest
+
+import csv
+
 from collections import Counter
 
-import axelrod
-import axelrod.interaction_utils as iu
 import pandas as pd
+
+from dask.dataframe.core import DataFrame
+
+from numpy import mean, nanmedian, std
+
+import axelrod as axl
+import axelrod.interaction_utils as iu
```

Comment:
Looking at the issue.  It looks like it was decide that only axelrod would be imported like this, and that you'd use axl.interaction_utils elsewhere.

Reply:
Yeah, that's right.